exercises/pythagorean-triplet/src/Triplet.hs

```diff
@@ -1,7 +1,10 @@
 module Triplet (isPythagorean, mkTriplet, pythagoreanTriplets) where
 
-isPythagorean = error "You need to implement this function."
+isPythagorean :: (Int, Int, Int) -> Bool
```

Comment:
I think we can consider redesigning this exercise without `mkTriplet`, @petertseng. The way it is now, I think it is over-specified.